.vscode/settings.json

```diff
@@ -519,7 +519,21 @@
         "source_location": "cpp",
         "*.ipp": "cpp",
         "ranges": "cpp",
-        "span": "cpp"
+        "span": "cpp",
```

Comment:
I keep getting this diff locally. I'm not sure if it's because I'm on mac? If someone can test that it doesn't break on linux / windows, I'd be inclined to leave it.

Reply:
This is just VSCode writing down things about the headers it's seen. This won't break anything.